packages/graphics/src/Graphics.js

```diff
@@ -322,7 +322,7 @@ export class Graphics extends Container
         options = Object.assign({
             width: 0,
             texture: Texture.WHITE,
-            color: 0x0,
+            color: 0xFFFFFF,
```

Comment:
What is this white? Black was intentional as a default.

Reply:
Because it is tint for texture, and always was white. 
Black tint => black filled result, texture not showing.

Reply:
Ohhh, I didn't realized that `color` was a tint. The documentation needs to be updated.

Reply:
oh, but in lineStyle default was black, right? eh, we might have a problem, need one more test